website/cue/reference/remap/functions.cue

```diff
@@ -36,6 +36,7 @@ remap: {
 		"Coerce",
 		"Convert",
 		"Debug",
+		"Enrichment",
```

Comment:
Nit: we might call this `External Data` instead, and chuck  `get_env_var` into this category as well.

Reply:
I think I like keeping functions that specifically interact with enrichment tables in their own section, but I don't have strong opinions.

Reply:
That is a fair point. I didn't think we'd get any other than the two we have now, but that might not be the case. We can go with the current set-up, and see if changes are needed in the future.